docs/source/training/training.rst

```diff
@@ -0,0 +1,787 @@
+         ``(0,2.5), (2,3), (3,1), (5,3), (6,2)``
+      2. A linear tendency starting from 2.5, with a rate of change of 0.25, lasting 3 seconds.
+
+      .. hint::
+         Detailed descriptions of the tendencies can be found :ref:`here <available-tendencies>`.
+
+   .. md-tab-item:: Solution
+
+
+      Your waveform can contain for example the following tendencies:
+
+      .. code-block:: yaml
+
+         - {type: piecewise, time: [0, 2, 3, 5, 6], value: [2.5, 3, 1, 3, 2]}
+         - {type: linear, from: 2.5, rate: 0.25, duration: 3}
+
+      You should see the following waveform:
+
+      .. image:: ../images/training/complex.png
+         :align: center
+
+Exercise 3b: Smoothing
+^^^^^^^^^^^^^^^^^^^^^^
+
+.. md-tab-set::
+   .. md-tab-item:: Exercise
+
+      Continuing from the waveform in the previous exercise, modify it to include a 
+      **smooth** tendency with a duration of 1 between the two tendencies. What do you notice?
+
+   .. md-tab-item:: Solution
+      
+      Your waveform can contain for example the following tendencies:
+
+      .. code-block:: yaml
+
+         - {type: piecewise, time: [0, 2, 3, 5, 6], value: [2.5, 3, 1, 3, 2]}
+         - {type: smooth, duration: 1}
+         - {type: linear, from: 2.5, rate: 0.25, duration: 3}
+
+      .. image:: ../images/training/smooth.png
+         :align: center
+
+      You should see the following waveform. Notice how the smooth tendencies ensure 
+      continuous value and derivative across multiple tendencies.
+
+
+
+
+Exercise 3c: Repeating Waveforms
+^^^^^^^^^^^^^^^^^^^^^^^^^^^^^^^^
+
+.. md-tab-set::
+   .. md-tab-item:: Exercise
+
+      You can create repeating patterns using the ``repeat`` tendency. This tendency 
+      allows you to specify the ``waveform`` parameter, here you can provide a list of 
+      tendencies that will be repeated.
+
+      Copy the tendency list from the previous exercise use a ``repeat`` tendency to make it repeat three times.
+      Ensure that the end of the linear tendency and the start of the piecewise tendency also smoothly 
+      transition into each other, in 1 second.
+
+   .. md-tab-item:: Solution
+
+      A smooth tendency was added as a last tendency to smoothly transition from the 
+      linear tendency back into the piecewise linear tendency. This whole waveform is 
+      placed in the ``waveform`` parameter of the repeat tendency. Since the tendencies
+      combine up to a total length of 11 (6+1+3+1), the total ``duration`` of the repeat
+      tendency is set to 33, to obtain three full cycles.
+
+      .. code-block:: yaml
+
+         - type: repeat
+           duration: 33
+           waveform:
+           - {type: piecewise, time: [0, 2, 3, 5, 6], value: [2.5, 3, 1, 3, 2]}
+           - {type: smooth, duration: 1}
+           - {type: linear, from: 2.5, rate: 0.25, duration: 3}
+           - {type: smooth, duration: 1}
+
+      You should see the following waveform:
+
+      .. image:: ../images/training/repeat.png
```

Comment:
As discussed yesterday, this looks pretty chaotic / noisy.

The smooth transition at the end of the repeated waveform still looks quite jagged (due to the relatively large mismatch in values)